implementation/registry-of-identifiers/src/main/java/eu/efti/identifiersregistry/IdentifiersMapper.java

```diff
@@ -1,29 +1,66 @@
 package eu.efti.identifiersregistry;
 
 import eu.efti.commons.dto.IdentifiersDto;
-import eu.efti.identifiersregistry.entity.Identifiers;
+import eu.efti.commons.dto.TransportVehicleDto;
+import eu.efti.identifiersregistry.entity.Consignment;
+import eu.efti.identifiersregistry.entity.MainCarriageTransportMovement;
+import eu.efti.identifiersregistry.entity.UsedTransportEquipment;
 import lombok.RequiredArgsConstructor;
-import org.modelmapper.ModelMapper;
 import org.springframework.stereotype.Component;
 
-import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.List;
 
 @Component
 @RequiredArgsConstructor
 public class IdentifiersMapper {
 
-    private final ModelMapper mapper;
```

Comment:
Im guessing its because you havent uplated the interfaces yet so automatic mapping was complicated? Do you plan on using it back when you are finished?